gorfc/gorfc.go

```diff
@@ -278,7 +277,7 @@ func nWrapString(uc *C.SAP_UC, length C.int, strip bool) (result string, err err
 	}
 	utf8Size := C.uint(length*3) + 1
 	utf8str := (*C.char)(unsafe.Pointer(C.malloc((C.size_t)(utf8Size))))
-	//defer C.free(unsafe.Pointer(utf8str)) // _todo: Memory access error on Windows only, when trying to free RFCCHAR1 of RFCTABLE in function call test
+	defer C.free(unsafe.Pointer(utf8str)) // _todo: Memory access error on Windows only, when trying to free RFCCHAR1 of RFCTABLE in function call test
```

Comment:
da es sich bei der Variable um eine C Heap handelt, wird der Speicherbereich nicht vom Golang GC bereinigt. Weil das Package unter Windows eh als "experimental" gekennzeichnet ist, sollte die offen verfügbare Version zumindest unter Linux "out of the box" funktionieren, daher diese Änderung.